examples_tests/23.Autoexposure/main.cpp

```diff
@@ -4,20 +4,771 @@
 
 #define _NBL_STATIC_LIB_
 #include <nabla.h>
-#include <iostream>
-#include <cstdio>
 
+#include "../common/CommonAPI.h"
 
+#include "nbl/ext/LumaMeter/CLumaMeter.h"
 #include "nbl/ext/ToneMapper/CToneMapper.h"
-
-#include "../common/QToQuitEventReceiver.h"
+#include "nbl/ext/FullScreenTriangle/FullScreenTriangle.h"
 
 using namespace nbl;
-using namespace nbl::core;
-using namespace nbl::asset;
-using namespace nbl::video;
 
+class AutoexposureSampleApp : public ApplicationBase
+{
+	constexpr static uint32_t WIN_W = 1280u;
+	constexpr static uint32_t WIN_H = 720u;
+	constexpr static uint32_t SC_IMG_COUNT = 3u;
+	constexpr static uint32_t FRAMES_IN_FLIGHT = 5u;
+	static constexpr uint64_t MAX_TIMEOUT = 99999999999999ull;
+	static_assert(FRAMES_IN_FLIGHT > SC_IMG_COUNT);
```

Comment:
ok so @nipunG314 this needs to be rewritten to use the same base classes as example 24 or something else that displays an image on screen with the FSTriangle extension